src/InternPool.zig

```diff
@@ -5985,6 +5986,8 @@ pub const Tag = enum(u8) {
         flags: Flags,
         owner_nav: Nav.Index,
         zir_index: TrackedInst.Index,
+        location_or_descriptor_set: u32,
+        descriptor_binding: u32,
```

Comment:
this is ugly and I'm not sure what the right thing to do is. haven't seen other examples of how large tagged unions are dealt with

Reply:
Keep as it is and wait for mlugg's suggestion i guess.